### ISSUE TITLE
Environment variables are not overriding the config -- alpine docker container

### ISSUE THREAD

**Issue by shokinn** [coderaiser/cloudcmd#179]
* **Version** (`cloudcmd -v`): v10.2.2
* **Node Version** `node -v`: v10.4.1
* **OS** (`uname -a` on Linux): Linux 9b74feee0022 4.15.0-22-generic #24-Ubuntu SMP Wed May 16 12:15:17 UTC 2018 x86_64 Linux
* **Docker Container**: coderaiser/cloudcmd:latest-alpine
* **Browser name/version**: Firefox 60.0.2
* **Used Command Line Parameters**:  
```
sudo docker run \
--name cloud_commander \
--volume /mnt:/mnt/media \
--env CLOUDCMD_NAME="TEST" \
--env CLOUDCMD_AUTH=false \
--env CLOUDCMD_USERNAME="" \
--env CLOUDCMD_PASSWORD="" \
--env CLOUDCMD_EDITOR="edward" \
--env CLOUDCMD_COLUMNS="name-size-date-owner-mode" \
--env CLOUDCMD_CONTACT=false \
--env CLOUDCMD_CONFIG_DIALOG=false \
--env CLOUDCMD_CONSOLE=false \
--env CLOUDCMD_SYNC_CONSOLE_PATH=false \
--env CLOUDCMD_TERMINAL=false \
--env CLOUDCMD_KEYS_PANEL=true \
--env CLOUDCMD_ROOT="/mnt" \
--env CLOUDCMD_VIM=false \
--env CLOUDCMD_CONFIRM_COPY=true \
--env CLOUDCMD_CONFIRM_MOVE=true \
--network some_br \
--ip 172.17.0.10 \
--user "$(sudo -u media id -u):$(sudo -u media id -g)" \
--detach \
coderaiser/cloudcmd:latest-alpine
```
* **Changed Config**: -

Environment variables does not override the config.
When I open the web page I'm forced to enter a username and a password even though.

**Comment by shokinn** [comment]
Workaround:
```shell
sudo mkdir -p /docker/appdata/cloudcom && \
cat << EOF | sudo tee /docker/appdata/cloudcom/.cloudcmd.json
{
    "name": "",
    "auth": false,
    "username": "",
    "password": "",
    "algo": "sha512WithRSAEncryption",
    "editor": "edward",
    "packer": "tar",
    "diff": true,
    "zip" : true,
    "buffer": true,
    "dirStorage": false,
    "online": true,
    "open": false,
    "keysPanel": true,
    "port": 8000,
    "ip": null,
    "root": "/mnt",
    "prefix": "",
    "progress": true,
    "contact": false,
    "confirmCopy": true,
    "confirmMove": true,
    "configDialog": false,
    "oneFilePanel": false,
    "console": false,
    "syncConsolePath": false,
    "terminal": false,
    "terminalPath": "",
    "showConfig": false,
    "vim": false,
    "columns": "name-size-date-owner-mode"
}
EOF
```

```shell
sudo docker run \
--name cloud_commander \
--volume /mnt/media:/mnt \
--volume /docker/appdata/cloudcom/.cloudcmd.json:/.cloudcmd.json \
--network nerdkino_br \
--ip 172.17.0.10 \
--user "$(sudo -u media id -u):$(sudo -u media id -g)" \
--detach \
coderaiser/cloudcmd:latest-alpine \
--no-terminal
```

**Comment by coderaiser** [comment]
Thank you :), fixed with 1330acb.

**Comment by coderaiser** [comment]
Landed in [v10.2.3](https://github.com/coderaiser/cloudcmd/releases/tag/v10.2.3).
